OneCommunityGlobal/HighestGoodNetworkApp
# added and removed margin from some component to fix the style

This PR is related to PR #809
  On previous PR, the header height was fixed so that there are no overlapping  and a gap between the header and the component below it. but some component do not require the gap on original design, so in this PR the styling is changed accordingly.
Before:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86232710/4f3de264-8d72-44ad-8fc7-26c03d03d26a

after:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86232710/b0374768-884f-45de-9d06-3ca840a0fcfb

How to test:
 1.  checkout to this branch
 2.  npm run start:local
 3.  sign in to any account preferably admin, and the pages should look like as shown in after video. also check the responsiveness.
